server/console.go

```diff
@@ -109,14 +111,10 @@ func StartConsoleServer(logger *zap.Logger, startupLogger *zap.Logger, db *sql.D
 	// grpcGatewayRouter.Handle("/", console.Handler()).Methods("GET")
 	// grpcGatewayRouter.PathPrefix("/assets/").Handler(console.Handler()).Methods("GET")
 
-	grpcGatewayRouter.HandleFunc("/rpcz", func(w http.ResponseWriter, r *http.Request) {
-		zpages.Handler.ServeHTTP(w, r)
-	})
-	grpcGatewayRouter.HandleFunc("/tracez", func(w http.ResponseWriter, r *http.Request) {
-		zpages.Handler.ServeHTTP(w, r)
-	})
-	grpcGatewayRouter.HandleFunc("/public/", func(w http.ResponseWriter, r *http.Request) {
-		zpages.Handler.ServeHTTP(w, r)
+	zpagesMux := http.NewServeMux()
```

Comment:
Shall we comment out zpages for the main code line?

Reply:
I'll look at this in a follow-up commit.